monitoring/statuscake/resources.tf

```diff
@@ -0,0 +1,43 @@
+resource "statuscake_uptime_check" "main" {
+  for_each = var.urls
+
+  name           = each.value
+  contact_groups = var.contact_groups
+  confirmation   = 2
+  trigger_rate   = 0
+  check_interval = 30
+  regions        = ["london", "dublin"]
+
+  http_check {
+    follow_redirects = true
+    timeout          = 40
+    request_method   = "HTTP"
+    status_codes     = ["204", "205", "206", "303", "400", "401", "403", "404", "405", "406", "408", "410", "413", "444", "429", "494", "495", "496", "499", "500", "501", "502", "503", "504", "505", "506", "507", "508", "509", "510", "511", "521", "522", "523", "524", "520", "598", "599"]
+    validate_ssl     = false
+  }
+
+  monitored_resource {
+    address = each.value
+  }
+}
+
+resource "statuscake_ssl_check" "main" {
```

Comment:
There may be a lot of duplication if we monitor multiple URLs per app. But SSL only requires 1 monitor.
It feels like this check would be better located in https://github.com/DFE-Digital/terraform-modules/tree/main/domains/environment_domains

Reply:
I think it would be better to change this module to accept a separate variable for the SSL check then. Having the StatusCake monitoring in `environment_domains` feels like we'll have the same problem with the application module where services without StatusCake need to set up the provider to be able to use it.

Reply:
OK makes sense. Maybe a simple `var.monitoring_tls_domains`?

Reply:
I've amended the commit, split the `urls` variable in to two.